### PR TITLE
Config changes to publish the package to NPM

### PR DESCRIPTION
**Pull request by nagarjun**
This PR introduces the following changes

* Added a GitHub Action to publish the package to NPM whenever a new release is created
* Updated the package.json file

### CODE REVIEW

File: README.md
Comment (nagarjun) [inline README.md:32]: @evanfuller @rssathe how do you guys feel about this name?

Comment (evanfuller) [reply README.md:32]: works for me

File: .github/workflows/publish-to-npm.yml
Comment (evanfuller) [inline .github/workflows/publish-to-npm.yml:18]: can you share this secret NPM_TOKEN with a couple of other folks so we have some redundancy?

**Review by evanfuller — APPROVED** [review]

File: .github/workflows/publish-to-npm.yml
Comment (nagarjun) [reply .github/workflows/publish-to-npm.yml:18]: Yeah, I was just waiting for the final decision on the NPM org name in our Slack conversation. Will share on Keybase once we finalise.

File: package.json
Comment (nagarjun) [reply package.json:20]: Lol.. valid point but, it's alright. I wouldn't mind a little spam in exchange for NPM fame. :-)

**Pull request merged**
